felles/abac/src/main/java/no/nav/vedtak/sikkerhet/abac/policy/InternBrukerPolicies.java

```diff
@@ -0,0 +1,150 @@
+            var beslutterLikSaksbehandler = Objects.equals(beskyttetRessursAttributter.getBrukerId(), saksbehandler);
+            return beslutterLikSaksbehandler ? Tilgangsvurdering.avslåGenerell("Beslutter er lik saksbehandler") : Tilgangsvurdering.godkjenn(AnsattGruppe.BESLUTTER);
+        }
+        // Overstyring
+        if (ForeldrepengerAttributter.VALUE_FP_BEHANDLING_STATUS_UTREDES.equals(behandlingStatus) &&
+            ForeldrepengerAttributter.VALUE_FP_SAK_STATUS_BEHANDLES.equals(fagsakStatus) &&
+            ForeldrepengerAttributter.VALUE_FP_AKSJONSPUNKT_OVERSTYRING.equals(overstyring)) {
+            return Tilgangsvurdering.godkjenn(AnsattGruppe.OVERSTYRER);
+        }
+        // Ordinær saksbehandling
+        if ((ForeldrepengerAttributter.VALUE_FP_BEHANDLING_STATUS_UTREDES.equals(behandlingStatus) || ForeldrepengerAttributter.VALUE_FP_BEHANDLING_STATUS_OPPRETTET.equals(behandlingStatus)) &&
+            (ForeldrepengerAttributter.VALUE_FP_SAK_STATUS_BEHANDLES.equals(fagsakStatus) || ForeldrepengerAttributter.VALUE_FP_SAK_STATUS_OPPRETTET.equals(fagsakStatus)) &&
+            !ForeldrepengerAttributter.VALUE_FP_AKSJONSPUNKT_OVERSTYRING.equals(overstyring)) {
+            return Tilgangsvurdering.godkjenn();
+        }
+        return Tilgangsvurdering.avslåGenerell("InternRessurs har ikke tilgang til å oppdatere fagsak");
+    }
+
+    private static Tilgangsvurdering applikasjonPolicy(BeskyttetRessursAttributter beskyttetRessursAttributter, AppRessursData appRessursData) {
+        if (!appRessursData.getFødselsnumre().isEmpty() || !appRessursData.getAktørIdSet().isEmpty()) {
+            return Tilgangsvurdering.avslåGenerell("Applikasjon-ressurs kan ikke ha personer / saker");
+        }
+        if (ActionType.READ.equals(beskyttetRessursAttributter.getActionType())) {
+            return veilederEllerSaksbehandler(beskyttetRessursAttributter);
+        } else {
+            return Tilgangsvurdering.avslåGenerell("InternBruker kan ikke utføre handling for ressurs");
+        }
+    }
+
+    private static Tilgangsvurdering ventefristPolicy(BeskyttetRessursAttributter beskyttetRessursAttributter) {
+        if (ActionType.UPDATE.equals(beskyttetRessursAttributter.getActionType())) {
+            return veilederEllerSaksbehandler(beskyttetRessursAttributter);
+        } else {
+            return Tilgangsvurdering.avslåGenerell("InternBruker kan ikke utføre handling for ressurs");
+        }
+    }
+
+    private static Tilgangsvurdering driftPolicy(BeskyttetRessursAttributter beskyttetRessursAttributter) {
+        if (ActionType.READ.equals(beskyttetRessursAttributter.getActionType()) || ActionType.CREATE.equals(beskyttetRessursAttributter.getActionType())) {
+            if (harGruppe(beskyttetRessursAttributter, AnsattGruppe.DRIFT)) {
+                return Tilgangsvurdering.godkjenn();
+            } else {
+                return Tilgangsvurdering.godkjenn(AnsattGruppe.DRIFT);
```

Comment:
Er det ikke feil å godkjenne hvis bruker ikke har gruppe? Samme gjelder oppgavestyrer

Reply:
Jeg kunne lagt inn en FORTSETT(gruppe) men beholdt GODKJENN(gruppe) i betydning krever gruppe.
Se i PepImpl

Reply:
Synes "godkjenn" her blir litt feil ord å bruke. For her er det ikke godkjent... men skal vurderes på et senere tidspunkt om er godkjent eller ei. Forsett(AnsattGruppe.DRIFT)  eller vurderSenere(AnsattGruppe.DRIFT)? Føler det er mer beskrivende og skaper mindre forvirring.

Reply:
Vi får tenke litt på det å heller ta en reformatering senere. Tenker det er greit for nå.